localplanning.services/src/components/Footer.astro

```diff
@@ -1,21 +1,47 @@
 ---
 import Container from "@components/Container.astro";
-import { Image } from 'astro:assets';
-import mhclgLogo from '/public/mhclg-logo.svg';
-import odpLogo from '/public//odp-logo.svg';
+import { Image } from "astro:assets";
+import mhclgLogo from "/public/mhclg-logo.svg";
+import odpLogo from "/public//odp-logo.svg";
+import { getCollection } from "astro:content";
+
+const legalPages = await getCollection("legal");
```

Comment:
By grabbing the collection and iterating over it, we ensure that any new pages are added to the footer automatically.